roles/manage_cnp/tasks/main.yml

```diff
@@ -0,0 +1,116 @@
+---
+# tasks file for role: manage_cnp
+# Options"
+# add-db, add-schema, add-role
+# drop-db, drop-schema, drop-role
+#- name: cnp_task not defined in 'supported_tasks' list
+#  ansible.builtin.fail:
+#    msg: cnp_task - "{{ cnp_task }}" is not in supported_tasks list "{{ supported_tasks }}"
+#  when: cnp_task not in supported_tasks
+- name: Validate cnp_task exist in supported_tasks list
+  ansible.builtin.fail:
+    #fail: msg="One of the defined cnp_task does not exist in the supported_tasks list"
+    msg: "cnp_task - {{ cnp_task | difference(supported_tasks) }} is not defined in supported_tasks"
+  when: cnp_task | difference(supported_tasks)
+
+- name: Validate cnpg_namespace
+  ansible.builtin.fail:
+    fail: msg="The variable cnpg_namespace is not defined"
+    msg: "The variable cnpg_namespace is not defined"
+  when:
+    - cnpg_namespace is not defined
+
+- name: Validate pod_name
+  ansible.builtin.fail:
+    fail: msg="The variable 'pod_name' is not defined"
+    msg: "The variable pod_name is not defined"
+  when:
+    - pod_name is not defined
+
+- name: Validate db_name
+  ansible.builtin.fail:
+    fail: msg="The variable 'db_name' is not defined"
+    msg: "The variable db_name is not defined"
+  when:
+    - db_name is not defined
+
+- name: Validate db_role
+  ansible.builtin.fail:
+    fail: msg="The variable 'db_role' is not defined"
+    msg: "The variable db_role is not defined"
+  when:
+    - db_role is not defined
+
+- name: Validate db_role_password
+  ansible.builtin.fail:
+    fail: msg="The variable 'db_role_password' is not defined"
+    msg: "The variable db_role_password is not defined"
+  when:
+    - db_role_password is not defined
+
+- name: Validate add_db_sql variable
+  ansible.builtin.fail:
+    fail: msg="The variable 'add_db_sql' is not defined"
+    msg: "The variable add_db_sql is not defined"
+  when:
+    - add_db_sql is not defined
```

Comment:
Is this a variable that is defined in defaults/main.yml? If so, I do not think it needs to be validated